### PR TITLE
2022-05-10-sbiobertresolve_cpt_procedures_measurements_augmented_en_3_0

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/Cabir40/2022-05-10-sbiobertresolve_cpt_procedures_measurements_augmented_en_3_0.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    .setOutputCol("word_embeddings")
+
+ner = MedicalNerModel.pretrained("ner_jsl", "en", "clinical/models") \
+    .setInputCols(["sentence", "token", "word_embeddings"]) \
+    .setOutputCol("ner")\
+
+ner_converter = NerConverterInternal()\
+    .setInputCols(["sentence", "token", "ner"])\
+    .setOutputCol("ner_chunk")\
+    .setWhiteList(["procedure", "test", "Imaging_Technique", "External_body_part_or_region", "Internal_organ_or_component"])
```
Comment (muhammetsnts) [inline docs/_posts/Cabir40/2022-05-10-sbiobertresolve_cpt_procedures_measurements_augmented_en_3_0.md:61]: Only Procedure and Test

File: docs/_posts/Cabir40/2022-05-10-sbiobertresolve_cpt_procedures_measurements_augmented_en_3_0.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+      .setOutputCol("word_embeddings")
+
+val ner = MedicalNerModel.pretrained("ner_jsl", "en", "clinical/models")
+      .setInputCols(Array("sentence", "token", "word_embeddings"))
+      .setOutputCol("ner")
+
+val ner_converter = new NerConverterInternal()
+      .setInputCols(Array("sentence", "token", "ner"))
+      .setOutputCol("ner_chunk")
+      .setWhiteList(Array("procedure", "test", "Imaging_Technique", "External_body_part_or_region", "Internal_organ_or_component"))
```
Comment (muhammetsnts) [inline docs/_posts/Cabir40/2022-05-10-sbiobertresolve_cpt_procedures_measurements_augmented_en_3_0.md:124]: Only Procedure and Test

File: docs/_posts/Cabir40/2022-05-10-sbiobertresolve_cpt_procedures_measurements_augmented_en_3_0.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+## Description
+
+This model maps medical entities to CPT codes using Sentence Bert Embeddings. The corpus of this model has been extented to measurements, and this model is capable of mapping both procedures and measurement concepts/entities to CPT codes. Measurement codes are helpful in codifying medical entities related to tests and their results.
+
+## Predicted Entities
+
+`CPT Codes`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
```
Comment (muhammetsnts) [inline docs/_posts/Cabir40/2022-05-10-sbiobertresolve_cpt_procedures_measurements_augmented_en_3_0.md:27]: If demo exists, please share the link here

**Review by muhammetsnts — CHANGES_REQUESTED** [review]

**Review by muhammetsnts — APPROVED** [review]

**Comment by Cabir40** [comment]
whitelist be updated (Precedures and Test)
added colab link
added demo link

**Pull request merged**